doc/rbd/index.rst

```diff
@@ -4,18 +4,24 @@
 
 .. index:: Ceph Block Device; introduction
 
-A block is a sequence of bytes (often 512).
+**rbd** is a utility that can manipulate rados block device (RBD) images. It is
```

Comment:
`rbd` utility (CLI tool) isn't used by Linux kernel or QEMU/KVM.  Users may use to create and manipulate RBD images but that is not the only way to do that.  Also, RADOS should be capitalized.